tests/src/python/test_qgsserver.py

```diff
@@ -475,6 +477,23 @@ def test_wms_getmap_basic(self):
         r, h = self._result(self.server.handleRequest(qs))
         self._img_diff_error(r, h, "WMS_GetMap_Basic")
 
+        qs = "&".join(["%s=%s" % i for i in list({
```

Comment:
This test file has really become too big, if you have time, would you please split the server tests into different files, one for each service? That would also help to manage test blacklisting in case we have a failing test.